content/en/docs/concepts/configuration/pod-overhead.md

```diff
@@ -26,31 +26,138 @@ on top of the container requests & limits.
-- in any custom API servers that use feature gates
-
-{{< note >}}
-Users who can write to RuntimeClass resources are able to have cluster-wide impact on
-workload performance. You can limit access to this ability using Kubernetes access controls.
-See [Authorization Overview](/docs/reference/access-authn-authz/authorization/) for more details.
-{{< /note >}}
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled (it is on by default as of 1.18)
+across your cluster, and a `RuntimeClass` is utilized which defines the `Overhead` field.
+
+### Usage example
+
+To use the `PodOverhead` feature, a `RuntimeClass` must be utilized which defines the `Overhead` field. As an example,
+the following `RuntimeClass` definition could be utilized for Kata Containers:
+
+```yaml
+---
+kind: RuntimeClass
+apiVersion: node.k8s.io/v1beta1
+metadata:
+    name: kata-fc
+handler: kata-fc
+overhead:
+    podFixed:
+        memory: "120Mi"
+        cpu: "250m"
+```
+
+Workloads which are created which specify the `kata-fc` RuntimeClass handler will take the memory and
+cpu overheads into account for resource quota calculations, node scheduling, as well as Pod cgroup sizing.
+
+Consider running the given example workload, test-pod:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  runtimeClassName: kata-fc
+  containers:
+  - name: busybox-ctr
+    image: busybox
+    stdin: true
+    tty: true
+    resources:
+      limits:
+        cpu: 500m
+        memory: 100Mi
+  - name: nginx-ctr
+    image: nginx
+    resources:
+      limits:
+        cpu: 1500m
+        memory: 100Mi
+```
+
+At admission time the RuntimeClass [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/)
+will update the workload's PodSpec to include the Overhead as described in the RuntimeClass. If the PodSpec already has this field defined,
+the Pod will be rejected. In the given example, since only the RuntimeClass name is specified, the Pod will be mutated to include the Overhead
+field within the PodSpec.
+
+After the RuntimeClass admission controller, the PodSpec will show:
+
+```bash
+$ kubectl get pod test-pod -o json | jq '.spec.overhead'
+{
+  "cpu": "250m",
+  "memory": "120Mi"
+}
+```
+
+If a ResourceQuota is defined, the sum of container requests as well as the
+Overhead field are counted.
+
+When kube-scheduler runs, the Overhead as well as sum of container requests are taken into account when identifying
+an appropriate node binding, again seeking a node which has 2.25 CPU and 320 MB of memory available.
+
+Once scheduled for the node, as part of the Pod creation process Kubelet will create a Pod cgroup (for non burstable QoS)
+and this will be sized as the sum of container (requests or limits) plus the Overhead defined in the PodSpec. 
+
+Taking a look at the container requests, we see a total of 2000m CPU and 200Mi:
```

Comment:
[Avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we)